.github/workflows/test_and_deploy.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        scipy-version: [">=0.16,<1.4", ">=1.4,<=1.6.2"]
+        scipy-version: [">=0.16,<1.4", ">=1.4,<=1.7.0"]
```

Comment:
Think line 66 also needs the same change?

Reply:
fixed in e671fbb947